packages/amplify-cli/src/init-steps/gen2Recommendation.ts

```diff
@@ -0,0 +1,34 @@
+import { $TSContext } from '@aws-amplify/amplify-cli-core';
+import { printer, prompter } from '@aws-amplify/amplify-prompts';
+import { isNewProject } from './s0-analyzeProject';
+
+export const gen2Recommendation = async (context: $TSContext): Promise<$TSContext> => {
+  if (!isNewProject(context)) {
+    return context;
+  }
+  printer.warn(
+    `AWS Amplify recommends using Amplify Gen 2 for new projects. Learn how to get started at https://docs.amplify.aws/react/start/quickstart/`,
```

Comment:
Unless this wording is coming from legal/product/somewhere we don't control, it sounds awkward to say "AWS Amplify recommends...". Instead I would just say
```suggestion
`Amplify Gen 2 is recommended for new projects. Learn how to get started at https://docs.amplify.aws/react/start/quickstart/`,
```

Reply:
Yes, the wording is from the ticket 🤦‍♀️

Reply:
Agree with the recommendation

Reply:
Done [22cff01](https://github.com/aws-amplify/amplify-cli/pull/13849/commits/22cff01e5e878bdf59bbb86ef5000c3fb21fef7e)